modin/backends/pandas/query_compiler.py

```diff
@@ -2631,10 +2672,13 @@ def compute_groupby(df):
             # determening type of raised exception by applying `aggfunc`
             # to empty DataFrame
             try:
-                agg_func(
-                    pandas.DataFrame(index=[1], columns=[1]).groupby(level=0),
-                    **agg_kwargs,
-                )
+                if isinstance(agg_func, dict):
```

Comment:
`pandas.DataFrame(index=[1], columns=[1]).agg(agg_func) if isinstance(
                    agg_func, dict
                ) else agg_func(
                    pandas.DataFrame(index=[1], columns=[1]).groupby(level=0),
                    **agg_kwargs,
                )`

Reply:
What is the reason to format everything into one line when black reformats it back into weird structures?
```python
                pandas.DataFrame(index=[1], columns=[1]).agg(agg_func) if isinstance(
                    agg_func, dict
                ) else agg_func(
                    pandas.DataFrame(index=[1], columns=[1]).groupby(level=0),
                    **agg_kwargs,
                )
```

Reply:
There is an opinion that ternary operator is faster than if/else block. We can keep if/else block when ternary operator is too complex to read.

Reply:
@YarShev Are you sure it is true about Python? I could imagine it happening in C/C++ when ternary operator is compiled into `cmov` instruction and there is no branching (BTW `cmov` is not always faster than branch if branch is predicted well). But in Python?